wowchemy/layouts/partials/navbar.html

```diff
@@ -139,6 +139,35 @@
     </div><!-- /.navbar-collapse -->
 
     <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
+      {{/* Look for social icons that should be displayed in the navbar.
+           For now just look for a superuser. If more than one superuser
+           is defined, run through all of them. */}}
+
+      {{ range where (where site.Pages "Section" "authors") ".Params.superuser" true }}
+        {{ range .Params.social }}
+          {{ if .display.navbar }}
+            {{ $pack := or .icon_pack "fas" }}
```

Comment:
All of this logic from $pack to $target is duplicated from another Wowchemy file. Let's try to follow engineering best practices and move this re-usable logic to a new function (e.g. `parse_social_link`) in the `functions/` dir.

Reply:
Done